<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorMapSettingEditWidget</class>
 <widget class="QWidget" name="ColorMapSettingEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_9">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_7" stretch="0,1,0">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0">
           <item>
            <widget class="QGroupBox" name="colorModeGroupBox">
             <property name="title">
              <string>Relationship between values and colors</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QRadioButton" name="colorModeAutoRadioButton">
                <property name="text">
                 <string>Automatically adjust based on value range</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="colorModeManualRadioButton">
                <property name="text">
                 <string>Setup Manually</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Expanding</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>10</width>
                  <height>0</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="valueRangeGroupBox">
           <property name="title">
            <string>Value Range</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QCheckBox" name="autoValueRangeCheckBox">
                <property name="text">
                 <string>Automatic</string>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0">
                 <widget class="QLabel" name="maxValueLabel">
                  <property name="text">
                   <string>Max:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="RealNumberEditWidget" name="maxValueEdit"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="minValueLabel">
                  <property name="text">
                   <string>Min:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="RealNumberEditWidget" name="minValueEdit"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="colorsGroupBox">
         <property name="title">
          <string>Colors</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QTableView" name="colorsTableView">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>350</width>
                <height>0</height>
               </size>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::AllEditTriggers</set>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::ContiguousSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QPushButton" name="addButton">
                <property name="text">
                 <string>&amp;Add</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="removeButton">
                <property name="text">
                 <string>&amp;Remove</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="Line" name="line">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="reverseButton">
                <property name="text">
                 <string>&amp;Reverse</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="equalDevisionButton">
                <property name="text">
                 <string>&amp;Equal division</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>17</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QCheckBox" name="fillLowerCheckBox">
              <property name="text">
               <string>Fill lower area</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="fillUpperCheckBox">
              <property name="text">
               <string>Fill upper area</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="switchModeButton">
           <property name="text">
            <string>Switch</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="legendGroupBox">
      <property name="title">
       <string>Color Bar Setting</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="ColorMapLegendSettingEditWidget" name="legendWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RealNumberEditWidget</class>
   <extends>QLineEdit</extends>
   <header>guibase/widget/realnumbereditwidget.h</header>
  </customwidget>
  <customwidget>
   <class>ColorMapLegendSettingEditWidget</class>
   <extends>QWidget</extends>
   <header>guicore/scalarstocolors/colormaplegendsettingeditwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>autoValueRangeCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxValueEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoValueRangeCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>minValueEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorModeManualRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>valueRangeGroupBox</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>77</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
